build.gradle

```diff
@@ -169,7 +169,7 @@ dependencies {
     integrationTestImplementation "org.testcontainers:kafka:$testcontainersVersion" // this is not Kafka version
     integrationTestImplementation "io.confluent:kafka-avro-serializer:$confluentPlatformVersion"
     integrationTestImplementation "io.confluent:kafka-connect-avro-converter:$confluentPlatformVersion"
-
+    integrationTestImplementation "org.awaitility:awaitility:4.1.1"
```

Comment:
[4.2.0](https://mvnrepository.com/artifact/org.awaitility/awaitility/4.2.0) ?

Reply:
You gotta leave a little something for dependabot to do! :D (Good catch)

Reply:
Fixed